examples/polypharmacy_gripnet/main.py

```diff
@@ -1,73 +1,60 @@
-import argparse
-import os
+import warnings
 
+import pytorch_lightning as pl
 import torch
 from config import get_cfg_defaults
-from model import GripNet
-from trainer import Trainer
+from model import GripNetLinkPrediction
+from utils import get_all_dataloader, load_data, setup_supervertex
```

Comment:
We do not put this into example typically. See other pl examples in PyKale and https://pykale.readthedocs.io/en/latest/tutorial.html